DNN Platform/Library/Entities/Urls/RewriterUtils.cs

```diff
@@ -35,7 +35,7 @@ namespace DotNetNuke.Entities.Urls
 {
     public class RewriterUtils
     {
-        internal static readonly Regex ServicesFrameworkRegex = new Regex("DesktopModules/.+/API/", RegexOptions.Compiled);
+        internal static readonly Regex ServicesFrameworkRegex = new Regex("desktopModules/.+/api/|api/.+/action/", RegexOptions.Compiled);
```

Comment:
Why this "/API/.+/Action" ? Why not "/API/Action/" ? The latter would be faster to match against.